### DISCUSSION TITLE
Group License with Kubernetes build agents

### DISCUSSION THREAD

**Discussion by noahmuney** [dashingsoft/pyarmor#1507] [General]
I have a kubernetes cluster made up of 5 host machines, and my CI creates new pods in the cluster every time it runs and destroys them when it is done.
I currently use pro license and connect to the internet for every run in order to register pyarmor.
Would it be possible to use the group license in order to avoid connecting to the internet?
Is there a way to do this without copying files from the host for every run? (as detailed in section 2.9.3.5 of the documentation)

**Comment by jondy** [comment]
Group license may not work direclty with default runner, but it may work with something like self-host runner, please check Kubernetes documentation.
I'm not expert on docker container, the example in section 2.9.3.5 just show how to work, please check docker documentation for more solutions.
